src/index.css

```diff
@@ -55,54 +55,122 @@ a:-webkit-any-link {
   text-decoration: none;
   color: #9c9c9c;
   cursor: pointer;
-  font-size: 12px;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
 }
 
 a:focus {
-  color: black;
+  color: #121212;
 }
 
 .icon {
   border: 1px solid #ebebeb;
   border-radius: 50%;
-  width: 24px;
-  height: 24px;
-  padding: 5px;
+  width: 2.813rem;
+  height: 2.813rem;
+  padding: 10px;
   color: #0290ff;
   cursor: pointer;
 }
 
-.text-1,
-.text-3 {
-  font-family: Arial, Helvetica, sans-serif;
-  font-size: 9px;
+.text-1 {
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
 }
 
 .text-3 {
-  color: #59a1e0;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
 }
 
 .text-2 {
-  font-family: Georgia, 'Times New Roman', Times, serif;
-  font-size: 11px;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.375rem;
   font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
As these three lines are getting repeated in different classes. The best practice is to create a new class and place these properties there instead of repeating the properties over and over again. It'll save your CSS lines as well 👍🏻 😄 
#### For Example
```css
.class-name {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```